cinema/views.py

```diff
@@ -21,24 +23,45 @@
 )
 
 
-class GenreViewSet(viewsets.ModelViewSet):
+class GenreViewSet(
+    viewsets.GenericViewSet,
+    mixins.ListModelMixin,
+    mixins.CreateModelMixin,
+):
```

Comment:
What the reason of this changes in each view?

Reply:
because we only need some part of CRUD operations